astroquery/mpc/core.py

```diff
@@ -344,8 +345,7 @@ def get_ephemeris_async(self, target, *, location='500', start=None, step='1d',
                             proper_motion='total', proper_motion_unit='arcsec/h',
                             suppress_daytime=False, suppress_set=False,
                             perturbed=True, unc_links=False,
-                            get_query_payload=False,
-                            get_raw_response=False, cache=False):
+                            get_query_payload=False, cache=False):
```

Comment:
This one was removed rather than deprecated because the parameter was unused and had no effect on the method's behavior.  I'll add a comment in the change log.

Reply:
The same comment for `get_observatory_codes`.

Reply:
Mentioned these changes (and justification) in the change log.